### ISSUE TITLE
Return of the standard values of Shop Localization, Languages and Currencies after every Meteor server restart

### ISSUE THREAD

**Issue by jekaspekas** [reactioncommerce/reaction#2578]
### Expected behavior
Do not set the defaults from the file "\private\data\Shops.json" after each server reboot.

### Actual Behavior
Return of the standard values of Shop Localization, Languages and Currencies after every Meteor server restarted.

### Steps to Reproduce the Behavior

1. Start reaction
1. Change any Dashboard values of Shop Localization, Languages or Currencies. After restarting the Meteor server, these values will change to the default values from the file "\private\data\Shops.json"

I think the problem is in the file "\server\startup\load-data.js". Reaction.Import.flush(Shops) - without checking the existence of the store in the database, overwrites the data from the file "\private\data\Shops.json".

This decision is not elegant, but it works:

import { Meteor } from "meteor/meteor";
import { Shops } from "/lib/collections";
import { Logger, Reaction } from "/server/api";
import { Fixture } from "/server/api/core/import";

export default function () {
  /**
   * Hook to setup core additional imports during Reaction init (shops process first)
   */
  Logger.info("Load default data from /private/data/");

####  if (!Reaction.getShopId()) {

    try {
      Logger.debug("Loading Shop Data");
      Reaction.Import.process(Assets.getText("data/Shops.json"), ["name"], Reaction.Import.shop);
      // ensure Shops are loaded first.
      Reaction.Import.flush(Shops);
    } catch (error) {
      Logger.error(error, "Bypassing loading Shop default data");
    }

    // make sure the default shop has been created before going further
    while (!Reaction.getShopId()) {
      Logger.debug("Loading default shop, waiting until it's ready before moving on...");
      Meteor._sleepForMs(1000);
    }

    try {
      Logger.debug("Loading Shipping Data");
      Fixture.process(Assets.getText("data/Shipping.json"), ["name"], Reaction.Import.shipping);
    } catch (error) {
      Logger.error(error, "Bypassing loading Shipping default data.");
    }

    try {
      Logger.debug("Loading Product Data");
      Fixture.process(Assets.getText("data/Products.json"), ["title"], Reaction.Import.product);
    } catch (error) {
      Logger.error(error, "Bypassing loading Products default data.");
    }

    try {
      Logger.debug("Loading Tag Data");
      Fixture.process(Assets.getText("data/Tags.json"), ["name"], Reaction.Import.tag);
    } catch (error) {
      Logger.error(error, "Bypassing loading Tags default data.");
    }
    //
    // these will flush and import with the rest of the imports from core init.
    // but Bulk.find.upsert() = false
    //
    Fixture.flush();

####  } else {

    try {
      Logger.debug("Loading Shipping Data");
      Fixture.process(Assets.getText("data/Shipping.json"), ["name"], Reaction.Import.shipping);
    } catch (error) {
      Logger.error(error, "Bypassing loading Shipping default data.");
    }

    try {
      Logger.debug("Loading Product Data");
      Fixture.process(Assets.getText("data/Products.json"), ["title"], Reaction.Import.product);
    } catch (error) {
      Logger.error(error, "Bypassing loading Products default data.");
    }

    try {
      Logger.debug("Loading Tag Data");
      Fixture.process(Assets.getText("data/Tags.json"), ["name"], Reaction.Import.tag);
    } catch (error) {
      Logger.error(error, "Bypassing loading Tags default data.");
    }
    //
    // these will flush and import with the rest of the imports from core init.
    // but Bulk.find.upsert() = false
    //
    Fixture.flush();

####  }

}

### Versions
Node: 6.11.1
NPM: 3.10.10
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.4.0
Docker: 17.05.0-ce

**Comment by zenweasel** [comment]
I can confirm that this is happening. This has to be a recently (re)introduced bug.

**Comment by aaronjudd** [comment]
In addition, we should add a flag for the ENV commandline -> that just allows bypassing of the imports completely.

**Comment by zenweasel** [comment]
I tested this back to `v1.0.0` and it's still happening so my idea that this is a recently re-introduced bug seems to be false.

**Comment by zenweasel** [comment]
Not sure if this is related to the cause of this issue, but even with the entirety of `load-data.json` commented out `Shops` data is still being loaded. 🤔 

```
in the root directory of your application.
01:09:18.655Z  INFO Reaction: Modified 0 documents while importing 2 to Shops
01:09:18.661Z  INFO Reaction: Reaction Version: 1.4.1
01:09:18.662Z  INFO Reaction: Modified 0 documents while importing 86 to Packages
01:09:18.718Z  INFO Reaction: Modified 0 documents while importing 491 to Translations
01:09:19.151Z  INFO Reaction: Reaction initialization finished.
01:09:19.158Z  INFO Reaction: Modified 0 documents while importing 10 to Templates
01:09:19.159Z  INFO Reaction: Modified 0 documents while importing 42 to Translations
```

**Comment by jekaspekas** [comment]
You run the console command: `reaction reset` ???

**Comment by zenweasel** [comment]
pardon?

**Comment by jekaspekas** [comment]
In my case, changes to the file "\server\startup\load-data.js" did not change the boot logic without console command: `reaction reset`
P.S. Sorry for my English

**Comment by zenweasel** [comment]
Closed via #2627
